boards/xtensa/intel_adsp_cavs15/tools/lib/stream_desc.py

```diff
@@ -38,7 +38,7 @@ def copy(self, data, size):
             raise ValueError("Not enough buffer. allocated: %d requested: %d"
                              % (self.alloc_size, size))
         logging.debug("Copying Data to DMA buffer")
-        self.buf[:] = data[:]
+        self.buf[:size] = data[:size]
```

Comment:
is this related or needed? Was the original version wrong?

Reply:
The original version wrong, it can only assign sequence of same size, buffer and data not same size, so need this change.

Reply:
it changed now because you removed the manifest from the image?

Reply:
This change is not due to remove manifest. Even if add manifest, the buffer and data size still not same, it is fix for load firmware process.

Reply:
ok, so it isn't really related to the main topic of this commit...

Reply:
yes, should I use a new commit for this change?

Reply:
I think that would be easier to understand, yes. Thanks!

Reply:
done, do you think it's ok?